src/Omnipay/Realex/RemoteGateway.php

```diff
@@ -25,7 +25,10 @@ public function getDefaultParameters()
             'merchantId' => '',
             'account'    => '',
             'secret'     => '',
-            '3dSecure'   => 0
+            '3dSecure'   => 0,
+            'authEndpoint'=>'https://epage.payandshop.com/epage-remote.cgi',
+            'secureDataVaultEndpoint'=>'https://epage.payandshop.com/epage-remote-plugins.cgi',
+            '3DSecureEndpoint'=>'https://epage.payandshop.com/epage-3dsecure.cgi'
```

Comment:
@3dgroup - just being picky, but can you match the case of this parameter to the existing 3dSecure one - ie. lowercase "d". It's just easier for other devs to remember then if it's consistent.